Source/WebKit/UIProcess/Cocoa/XPCConnectionTerminationWatchdog.mm

```diff
@@ -38,7 +38,11 @@
 }
     
 XPCConnectionTerminationWatchdog::XPCConnectionTerminationWatchdog(AuxiliaryProcessProxy& process, Seconds interval)
+#if ENABLE(CLOSE_XPC_CONNECTION_POST_LAUNCH)
+    : m_processID(process.processID())
```

Comment:
The previous code was working even if the process was still launching. The new code doesn't. Sounds like this may be an issue.

Reply:
Ah, good catch. I will try to address this.

Reply:
Ah, good point! I believe this issue has been addressed in the latest patch, where an audit token is used instead to avoid PID races.